CODE_OF_CONDUCT.md

```diff
@@ -2,12 +2,18 @@
 
 CocoaPods strongly values contributors from anywhere, regardless of gender, sexual orientation, disability, physical appearance, body size, race, or religion. As a result, the CocoaPods team has agreed to and enforces this code of conduct in order to provide a harassment-free experience for everyone who participates in the development of CocoaPods.
 
+CocoaPods also expects PG discussion on the issue tracker. Any use of explicit language is strictly prohibited.
```

Comment:
Will "PG" make sense to readers outside the USA? How about "expects polite discussion"